scripts/copy_existing_image_mapping.py

```diff
@@ -0,0 +1,55 @@
+import typer
+from prediction_market_agent_tooling.config import APIKeys
+from prediction_market_agent_tooling.gtypes import Wei, private_key_type
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.markets.omen.omen_contracts import (
+    OmenThumbnailMapping,
+)
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+from web3 import Web3
+
+
+def main(
+    private_key: str,
+    from_address: str,
+    to_address: str,
+) -> None:
+    """
+    Copy all existing image mapping on markets created by `private_key`, from OmenThumbnailMapping contract on `from_address` to OmenThumbnailMapping contract on `to_address`.
+    """
```

Comment:
Would you mind moving this functionality to PMAT? 
I'm not OK with having so much Python boilerplate within a foundry project - Foundry has its own scripting language and here we are making the project a bit too complex.
If upgradeability becomes an issue, there are other ways to address this (e.g. ProxyPattern), but for now I think one script inside PMAT should be enough for our use-case. Wdyt?

Reply:
Well this is related to this contract, not to tooling. Other contract repos are doing this as well (https://github.com/gnosis/pm-contracts -- 44% javascript, 1% python). Don't really see a problem with having some Python CI here (anyone not contributing Python code to this repo doesn't have to care about it).

But I'm okay with being outvoted once Evan is back, good thing there are 3 of us 😄

Reply:
Sure, but look at the Boilerplate required for the Python configuration (7 files for adding one Python script). I don't think that's a good idea - plus, this also becomes dependent on PMAT due to scripting (another place to update once versions comes out).
For those reasons I don't think this script belongs here - this is not how upgradability is solved on Solidity.

Reply:
> Boilerplate required for the Python configuration (7 files for adding one Python script). I don't think that's a good idea

What's problem with that? I don't see any issues arising from that. Except for the fact that we have many repos with the same boilerplate, so would be good to find a way to unify it somehow across them (because we will have new repos anyway).

Reply:
Standing corrected here, no need to have this here!

https://github.com/gnosis/prediction-market-agent-tooling/pull/291/files